### PR TITLE
Remove the Command Palette experiment docs

### PR DESCRIPTION
**Pull request by captainbrosset**
This PR removes the documentation for the **Command Palette** experimental feature, as the feature is being removed from Edge in version 123.
This PR defines a redirect, to make that URL redirect to What's New 122 > h2 section anchor about the removal of the experiment in Edge 123.  Defining the redirect was pending existence of that article section to link to, per [Comment](https://github.com/MicrosoftDocs/edge-developer/pull/3059#issuecomment-1943383986).

Live/Before: 
https://learn.microsoft.com/microsoft-edge/devtools-guide-chromium/experimental-features/edge-command-palette

Redirected URL to test in this PR:
https://review.learn.microsoft.com/microsoft-edge/devtools-guide-chromium/experimental-features/edge-command-palette?branch=pr-en-us-3059
Result: **Success**; end up at:
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/whats-new/2024/03/devtools-123?branch=pr-en-us-3059#command-palette-experimental-feature-has-been-removed
which is equivalent to live link:
https://learn.microsoft.com/microsoft-edge/devtools-guide-chromium/whats-new/2024/03/devtools-123#command-palette-experimental-feature-has-been-removed

AB#48990401

### CODE REVIEW

**Comment by captainbrosset** [comment]
`File microsoft-edge/devtools-guide-chromium/experimental-features/edge-command-palette.md with URL /microsoft-edge/devtools-guide-chromium/experimental-features/edge-command-palette was deleted without redirection. To avoid broken links, add a redirection.`

Once we have a What's New entry that mentions the feature removal, I will add a redirect to it.

File: microsoft-edge/devtools-guide-chromium/experimental-features/edge-command-palette.md
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/experimental-features/edge-command-palette.md:1]: define a redirect for this removed .md file
@captainbrosset

**Review by mikehoffms — CHANGES_REQUESTED** [review]
define a redirect for the removed .md file

**Review by mikehoffms — APPROVED** [review]
Approved PR after tested the redirect.

**Pull request merged**